pkg/util/iptables/iptables.go

```diff
@@ -783,3 +789,76 @@ func isResourceError(err error) bool {
 	}
 	return false
 }
+
+type ParseError interface {
+	Line() int
+	Error() string
+}
+
+type parseError struct {
+	cmd  string
+	line int
+}
+
+func (e parseError) Line() int {
+	return e.line
+}
+
+func (e parseError) Error() string {
+	return fmt.Sprintf("%s: input error on line %d: ", e.cmd, e.line)
+}
+
+type LineData struct {
+	Line int
```

Comment:
Needs a godoc comment, especially whether the first line is 0 or 1

Reply:
Done